pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -977,7 +978,7 @@ func requiresTopicInValue(s Sink) bool {
 	return s.getConcreteType() == sinkTypeWebhook
 }
 
-func makeChangefeedJobDescription(
+func makeChangefeedDescription(
```

Comment:
any reason why you didnt make the change to the comment on line 993 like in the original pr?

Reply:
Ah good catch, I must've missed it while resolving the merge conflict.